source/documentation/deploying_services/index.md

```diff
@@ -1008,6 +1005,102 @@ You must bind your app to the Redis service to be able to access the cache from
 
     If your app writes service connection errors to `STDOUT` or `STDERR`, you can view recent errors with ``cf logs APPNAME --recent``. See the section on [Logs](#logs) for details.
 
+### Connect to a Redis service instance from your local machine
+
+You must use the [Conduit](#conduit) plugin and stunnel tool to connect your local machine to your Redis service instance.
+
+#### Prerequisites
+
+You must:
+
+- install the Redis command line (CLI) tool on your local machine (this is included in the [standard Redis installation](https://redis.io/download) [external link])
+- [log into Cloud Foundry](/#setting-up-the-command-line)
+- [target the space](/#setting-a-target) where your Redis service instance is located
+
+#### Install and configure Conduit
+
+1. Run the following command to install the Conduit plugin:
+
+    ```
+    cf install-plugin conduit
+    ```
+
+1. Start Conduit by running:
+
+    ```
+    $ cf conduit SERVICE_NAME
+    ```
+
+    where `SERVICE_NAME` is a unique descriptive name for this service instance.
+
+    You will see the following output:
+
+    ```
+    service: SERVICE_NAME (redis)
+    host: 127.0.0.1
+    port: 7081
+    username:
+    password: REDIS_PASSWORD
+    db: DATABASE
+    ```
+
+    where `REDIS_PASSWORD` is the password given to you by Conduit.
+
+#### Install and configure stunnel
+
+1. Run the following command to install stunnel:
+
+    ```
+    brew install stunnel
+    ```
+
+    If you're not on a Mac, or don't have Homebrew, see the [stunnel
+    website](https://www.stunnel.org/downloads.html) [external link] for other
+    installation options.
+
+1. Create a stunnel configuration file that contains the following text:
+
+    ```ini
+    debug=7
+    options=NO_SSLv2
+    options=NO_SSLv3
+    foreground=yes
+    [redis]
+    client = yes
+    accept = 127.0.0.1:6379
+    connect = 127.0.0.1:7081
+    ```
+
+    where:
+    - `127.0.0.1:6379` is the default Redis host and port that stunnel will open (you will need to use a different port if you are already running a local Redis service instance)
+    - `127.0.0.1:7081` is the default Conduit host and port outputted by the `cf conduit SERVICE_NAME` command
+
+#### Connect your local machine to your Redis service instance
+
+1. Start stunnel by running:
+
+    ```
+    sudo stunnel CONFIGURATION_FILE
```

Comment:
Why does it require `sudo`?

Reply:
Because I blindly copied the AWS docs, which were blindly copied from some user. If you set a pidfile, you don't need sudo I think. Just testing.

Reply:
New commit added.